sycl/doc/extensions/SPIRV/SPV_INTEL_fp_fast_math_mode.asciidoc

```diff
@@ -0,0 +1,101 @@
+SPV_INTEL_FP_FAST_MATH_MODE
+===========================
+
+== Name Strings
+
+SPV_INTEL_fp_fast_math_mode
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
```

Comment:
Since we are not pushing this extension into Khronos Registry, would it be better to put URL of intel/llvm repo here instead?